tests/unit/mixins/Registry.ts

```diff
@@ -57,7 +57,7 @@ registerSuite({
 			class IntegrationTest extends TestWithRegistry {
 				render() {
 					return v('div', [
-						w('test', { id: `${Math.random()}` })
+						w<any>('test', { id: `${Math.random()}` })
```

Comment:
For reference these are the changes I needed to make for the examples: https://github.com/dojo/examples/pull/210